Sources/Player/Types/TrackingRegistration.swift

```diff
@@ -0,0 +1,10 @@
+//
+//  Copyright (c) SRG SSR. All rights reserved.
+//
+//  License information is available from the LICENSE file.
+//
+
+struct TrackingRegistration {
+    let type: Any.Type
```

Comment:
```suggestion
    let type: any PlayerItemTracker.Type
```

As the `TrackingRegistration` is always related to a `PlayerItemTracker`, we might be more specific by using the suggestion above. Using `Any` seems too generic, in my humble opinion.